### PR TITLE
Removed _id suffixes, specifed that time should be specified as tersely as possible.

### PR DESCRIPTION
**Pull request by RobertPincus**
This PR addresses #20 and #21 , changing meta-data from the form XX_id -> XX and making explicit that time need only be specified sufficiently to disambiguate files

### CODE REVIEW

File: naming_conventions.md [hunk 1]
```diff
@@ -4,36 +4,36 @@ This document suggests some standards for the treatment of EUREC4A metadata.  Th
...
+  * **product:**  An optional identifier to be used with the `instrument` to allow groups the flexibility to group measurements in different ways, i.e., SAFIRE_Aerosol might designate a bundle of aerosol instruments run by SAFIRE.
+  * **variable:** A specific quantity. Usually this will be the output from a sensor, but it could also be a derived or composite variable.  Examples might be a voltage on a power supply, a temperature reading, a radar reflectivity, the latitude and longitude of a track, or maybe the divergence from a sounding circle. It should follow a standard vocabulary whenever possible, i.e., NetCDF Climate and Forecast (CF) Metadata Conventions.
+  * **time:** For some data it may be useful to indicate the time of the measurement or the time-range of the data included in the file. This will adopt the form yyyymmddTHHMMSS-yyyymmddTHHMMSS, or e.g., yyyymmdd for a single instance. `time` should be specified at the minimum level of complexity required to distinguish one file from another and should be specified in UTC.
```
Comment (d70-t) [inline naming_conventions.md]: I am wondering if that statement is sufficient for everyone to understand what we want. Maybe we should add an example to it (i.e. if there is only one dataset per day, only `yyyymmdd` should be specified, if there is one dataset every hour, prefer to use `yyyymmddTHH`).

File: naming_conventions.md [hunk 2]
```diff
@@ -4,36 +4,36 @@ This document suggests some standards for the treatment of EUREC4A metadata.  Th
...
+ * **version:** The data versioning, in the form vN.M with N and M integers. Using version 0 for preliminary data is encouraged.
```
Comment (d70-t) [inline naming_conventions.md:15]: Following the discussion in #21, we should maybe add that the reason to include a version is that no two different files with the same name or maybe  "way of referencing" should exist. So it is ok to omit the version if another, clearly communicated way of referencing a specific revision of the dataset exists (e.g. git, DOI, etc...). And also if a version is part of the file name, the version **must always** be changed if the dataset content is changed.

**Review by d70-t — COMMENTED** [review]
I generally like the change. I propose to add a little bit more detail in two places.

File: naming_conventions.md (hunk 2 above)
Comment (RobertPincus) [reply naming_conventions.md:15]: @d70-t Language around time specification has been clarified with examples, and to say that a single time represents the starting window. 

For the moment let's leave the version number as part of the file name, in which case it's clear that the number should be changed if the data are updated. 

If you agree you could merge.

**Pull request merged**